.github/workflows/main.yml

```diff
@@ -1,12 +1,18 @@
-name: Validate and Release connector
+name: CI status
 on:
   pull_request:
   push:
     branches:
       - main
   release:
     types: [published]
+  workflow_dispatch:
+  schedule:
+    - cron: '0 4 * * *'
```

Comment:
Does it make sense that we run this every day when we already run on PRs and merges?

Reply:
I think so. It is about to feed the downstream [Build Status](https://cratedb.com/docs/crate/clients-tools/en/latest/status.html) page and [GitHub Actions](https://rapporto.readthedocs.io/source/github.html#actions-report) reports, that notify us about side effects happening on spots of the time axis other than on-PR, or on-merge.

While, in this case, it would not notify us about _functionality_ going south, but merely about _package building_ going south, I think it still has value, because also this little infrastructure can break.

Maybe it's utter nonsense, still after my explanation, then please let me know about it, and I will remove that cron configuration again.

Reply:
Oh I understand now, didn't think of that, it makes sense, thanks for the explanation!!